src/test/groovy/edu/ie3/io/factory/FactorySpecification.groovy

```diff
@@ -0,0 +1,12 @@
+package edu.ie3.io.factory
+
+import spock.lang.Specification
+import tec.uom.se.quantity.Quantities
+
+import javax.measure.Unit
+
+class FactorySpecification extends Specification {
```

Comment:
This looks like a Specification which is meant to be the abstract class for Spocks. 
To avoid confusion I would suggest to change it to a trait `trait FactoryTestHelper`  without the `Specification` extension, move it to `edu.ie3.test.helper` and extend the implementing subclasses with Specification. 

This has several advantages:
- no confusion on its purpose 
- no extra execution of this class by the test framework (which atm automatically executes it)
- clear structure that this is a helper 

Drawbacks:
- you need to extend all subclasses with `Specification` specifically (but this is IMHO also more an advantage than a disadvantage)

Reply:
Good call!